### PR TITLE
refresh the way to import rxjs operators and Subject.

### PR DESCRIPTION
**Pull request by Frodez**
#5470

### CODE REVIEW

**Comment by floreks** [comment]
Just read the bot comment and sign the cla as required.

**Comment by Frodez** [comment]
> Just read the bot comment and sign the cla as required.

Ok, I have already signed it.

File: src/app/frontend/common/services/guard/auth.ts [hunk 1]
```diff
@@ -36,10 +36,12 @@ export class AuthGuard implements CanActivate {
             return this.router_.navigate(['login']);
           }
 
-          return Observable.of(true);
+          return of(true);
         }),
       )
-      .catch(() => {
+      .catch((_err, _caught) => {
+        console.error(_err);
```
Comment (floreks) [inline src/app/frontend/common/services/guard/auth.ts:43]: Remove console print here.

File: src/app/frontend/common/services/guard/login.ts [hunk 2]
```diff
@@ -36,7 +36,9 @@ export class LoginGuard implements CanActivate {
           return of(true);
         }),
       )
-      .catch(() => {
+      .catch((_err, _caught) => {
```
Comment (floreks) [inline src/app/frontend/common/services/guard/login.ts:39]: Same as above.

**Comment by Frodez** [comment]
> Just read the bot comment and sign the cla as required.

Oh sorry, I find that I insert some `console.log` in this PR. Could you modify them? I spent several hours for pre-commit procedure, and I don't want to try it again.

File: src/app/frontend/common/services/guard/auth.ts (hunk 1 above)
Comment (Frodez) [reply src/app/frontend/common/services/guard/auth.ts]: Ok I have removed it.

File: src/app/frontend/common/services/guard/login.ts (hunk 2 above)
Comment (Frodez) [reply src/app/frontend/common/services/guard/login.ts]: Ok I have removed it.

**Comment by floreks** [comment]
Thanks for your contribution.

/lgtm
/approve

**Pull request merged**